### DISCUSSION TITLE
Documentation's example not working.

### DISCUSSION THREAD

**Discussion by matheusfenolio** [panva/jose#303] [Q&A] [answered]
Hello! First of all, thanks for this awesome project. :smiley: 

I'm trying to generate a simple JWT token. I followed the example present in the documentation, but I guess I'm missing something. 

Code:
![image](https://user-images.githubusercontent.com/37229507/139278837-dc6e58c6-61aa-41ce-9e80-8b41106e8530.png)

Log:
![image](https://user-images.githubusercontent.com/37229507/139277107-f82152f7-13ab-42f0-8bcc-179c1139c389.png)

**Comment by panva** [comment, marked as answer]
https://github.com/panva/jose/issues/210

It's telling you the right thing. Uint8array is not a right input for an asymmetric algorithm.
